packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -1613,6 +1610,19 @@ class FocusTraversalGroup extends StatefulWidget {
   ///    bottom.
   final FocusTraversalPolicy policy;
 
+  /// If true, then the children of this group will be marked as not focusable.
+  ///
+  /// If any children of this group are currently focused when this is set to
+  /// true, they will lose focus.
+  ///
+  /// Defaults to false. Must not be null.
+  final bool excludeFocus;
```

Comment:
Should this use the same name, i.e. `overrideFocusability`?

Reply:
I considered that, but the difference is that there is no `canRequestFocus` on this API, so really all you can do is exclude focus.

Another option would be to change the semantics of `overrideFocusability` and rename it to be `excludeFocus` too, where it also means set `canRequestFocus` to false if it is set.  In fact... That sounds like a better API anyhow, since then you don't have to set two things to get it to exclude, and `overrideFocusability` is a no-op anyhow if `canRequestFocus` is true.  I'll see if I can make that work.

Reply:
True. Sounds like a good idea!